packages/@angular/cli/plugins/suppress-entry-chunks-webpack-plugin.ts

```diff
@@ -8,11 +8,19 @@ export class SuppressExtractedTextChunksWebpackPlugin {
     compiler.plugin('compilation', function (compilation: any) {
       // find which chunks have css only entry points
       const cssOnlyChunks: string[] = [];
-        const entryPoints = compilation.options.entry;
+      const entryPoints             = compilation.options.entry;
+      const regExp                  = new RegExp(
```

Comment:
There are many backslashes here. Have you considered RegExp literals (`//`) or ES6 template literals (` `` `)?